NextVersion.md

```diff
@@ -44,20 +44,25 @@ The following props are now required.
 - [@itwin/imodels-client-management](https://www.npmjs.com/package/@itwin/imodels-client-management)
 - [@itwin/reality-data-client](https://www.npmjs.com/package/@itwin/reality-data-client)
 
-#### New scopes required
+#### OIDC Scopes
+
+##### Required
 
 In your [iTwin App registration](https://developer.bentley.com/my-apps/), you'll need update your app to add the following scopes:
 
 - `imodels:read`
 - `realitydata:read`
 
-These scopes are no longer required:
+##### Removed
 
 - `openid`
 - `email`
 - `profile`
 - `organization`
 
+##### Optional
+
+- `users:read`
```

Comment:
maybe make a note that only if you intend to use the applicationInsightsKey, or something along those lines?

Reply:
dont need to make an extra req for this and dont need the extra scope anymore either